src/components/__tests__/LogInForm-test.tsx

```diff
@@ -57,15 +57,27 @@ describe("LogInForm", () => {
     expect(password.props().type).to.equal("password");
   });
   it("should optionally display extra fields", () => {
-    let extraText = <input type="text" key="extraText" name="extraText"/>;
-    let extraCheckbox = <input type="checkbox" key="extraCheckbox" name="extraCheckbox" aria-checked={false} />;
-    wrapper.setProps({ extraFields: [extraText, extraCheckbox] });
+    let extraText = (
+      <input
+        type="text"
+        key="extraText"
+        name="extraText"
+        placeholder="some text"
+      />);
+    let extraText2 = (
```

Comment:
This is annoying but `tslint-react-a11y` has a bug where it thinks all inputs need to have a placeholder or default text but that's not true for checkboxes. So making this a text input for now.

Here's the issue: https://github.com/microsoft/tslint-microsoft-contrib/issues/749
`tslint-react-a11y` uses `tslint-microsoft-contrib` under the hood but maybe it hasn't been updated.